### PR TITLE
Add documentation for Lyric Integration

### PR DESCRIPTION
**Pull request by shellster**
## Proposed change

This documentation is to support the PR linked below.  It is documentation for Honeywell Lyric devices.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:
- This PR fixes or closes issue:

https://github.com/home-assistant/home-assistant/pull/31387

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/lyric.markdown [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+title: Honeywell Lyric
+description: Instructions on how to integrate Honeywell Lyric thermostats within Home Assistant.
+logo: honeywell.png
+ha_category:
+  - Climate
+ha_release: 0.106.2
```
Comment (klaasnicolaas) [inline source/_integrations/lyric.markdown]:
```suggestion
ha_release: 0.107
```

File: source/_integrations/lyric.markdown [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+### Configuration File
+
+```yaml
+# Example configuration.yaml entry
+lyric:
+  client_id: CLIENT_ID
+  client_secret: CLIENT_SECRET
+```
+
+Configuration variables:
```
Comment (klaasnicolaas) [inline source/_integrations/lyric.markdown]:
```suggestion
```

Please remove this line

File: source/_integrations/lyric.markdown [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+The `lyric` climate platform integrates Home Assistant with [Honeywell Home](https://developer.honeywellhome.com/) (Lyric) climate systems.  
+
+It uses the [python-lyric](https://github.com/bramkragten/python-lyric) client library. It does not currently support leak detector devices.
+
+If your setup thermostat cannot be added when you log into the [Honeywell Home](https://developer.honeywellhome.com/) website, your thermostat is _not_ supported by this component.  Perhaps you are looking for one of the other Honeywell modules instead. This module specifically supports and is tested with Lyric T5 and T6. Some people have also stated that it works with T9, but it hasn't been extensively tested.
+
+## Configuration
+
+### Setup Developer Account and Keys
```
Comment (klaasnicolaas) [inline source/_integrations/lyric.markdown:22]:
```suggestion
### Setup Developer Account and Keys

```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by whattheschnell** [comment]
Looks like node.js was having issues when you committed.  Have to re-run the Lint Markdown test or re-commit.  Shouldn't be any reason for it not to pass.

**Review by klaasnicolaas — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged 👍
⚠️ The ha_release might need an update before merging this one in.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**